robot_specific_config/abb_irb4600_support/urdf/irb4600_40_255_macro.xacro

```diff
@@ -17,6 +18,9 @@
         </geometry>
         <xacro:material_abb_graphite_white/>
       </visual>
+      <xacro:cylinder_inertial radius="0.275" length="0.18" mass="30.28">
```

Comment:
could you please add a comment explaining where these values came from? They seem 'precise', so I'm wondering if they came from a datasheet, or some other source.

For future reference it would be good to have a note of that somewhere.

If it is the same for all `length`s and `mass`es, perhaps the package manifest would be a good place to put a note.

Reply:
I would perhaps still suggest to -- in addition to this comment -- add something like [this](https://github.com/sam-xl/abb_ros2/blob/d2c4b8ac37584574805c82c16a223d0e19c140c3/robot_specific_config/abb_irb1200_support/package.xml#L21-L27) to the manifest.